config-connector/solutions/iam/kpt/storage-bucket-iam/README.md

```diff
@@ -0,0 +1,37 @@
+storage-bucket-iam
+==================================================
+
+# NAME
+
+  storage-bucket-iam
+
+# SYNOPSIS
+
+  Config Connector compatible yaml to enable permissions for a storage bucket.
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account with the following roles:
+
+  Note: Using uniform bucket-level access control is recommended for this package.
```

Comment:
What does it mean by "uniform bucket-level access control"?

Reply:
It's an option when creating the storage bucket. Without it each object within the bucket has its own access control list.

Reply:
How does it impact the usage of this kpt package? Can customers easily use the non-uniform bucket-level access control? I'm not sure why we want to mention it here.

Reply:
Mark the comment as blocking merge.

Reply:
The package will not work as well without uniform bucket-level access control because it doesn't manage individual objects within the bucket.